frontend/ui/lib/model/trip.dart

```diff
@@ -1,9 +1,18 @@
 import 'package:tripmeout/model/location.dart';
 
 class Trip {
-  Trip({this.name, this.id, this.location});
+  Trip({this.name = "", this.id = "", location})
```

Comment:
Can you mark these fields as nullable rather than giving them empty string values?

Reply:
Yeah, before they were nullable, I just prefer the empty string behavior over null. I'm not super opinionated though.